### PR TITLE
refactor: Remove unnecessary class.

### LINKED ISSUE #143: Remove unnecessary created class
**Actual Behaviour**

Multiple classes found for aboutus activity.

**Expected Behaviour**

Only one c;ass should function as about us activity.

**Would you like to work on the issue?**
Yes
Let us know if this issue should be assigned to you or tell us who you think could help to solve this issue.

### PR DESCRIPTION
**Pull request by chanmol1999**
Fixes #143 

**Changes**: deleted the unnecessary class. 

**Screenshot/s for the changes**:
Screenrecording_20190407_161821

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [ ] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [ x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [ ] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x ] My code does not contain any extra lines or extra spaces
- [x ] I have requested reviews from other members

**APK for testing**: [Compress the `app-debug.apk` file into a `<feature>.rar` or `<feature>.zip` file and upload it here]

### CODE REVIEW

**Comment by chanmol1999** [comment]
Please review @jddeep @iamareebjamal

**Comment by adityastic** [comment]
Change your title from fix to refactor

**Comment by iamareebjamal** [comment]
https://duckduckgo.com/?q=refractor&t=canonical&ia=definition

**Comment by adityastic** [comment]
> https://duckduckgo.com/?q=refractor&t=canonical&ia=definition

My bad, refactor*

**Review by adityastic — APPROVED** [review]

**Comment by adityastic** [comment]
@iamareebjamal Please Review

File: app/src/main/res/layout/activity_about__us.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="@drawable/hill_background"
-    tools:context=".AboutUsActivity">
+    tools:context=".About_Us">
```
Comment (iamareebjamal) [inline app/src/main/res/layout/activity_about__us.xml:8]: Rename this class to AboutUsActivity

**Review by suresh-gandhi — COMMENTED** [review]
@chanmol1999 The changes are now included in [this](https://github.com/fossasia/neurolab-android/pull/183) PR.

**Pull request closed without merging**